docs/macros.py

```diff
@@ -69,41 +70,23 @@ def get_object_from_reference(reference):
     return module
 
 
-def parse_doc_string(doc):
-    lines = doc.split("\n")
-
-    summary, risk, fix_text, fix_code = "", "", "", ""
-    summary_complete, risk_complete, fix_complete = False, False, False
-
-    for line in lines:
-        if line.startswith("Risk:"):
-            summary_complete = True
-            continue
+def regex_for_splitting_paragraphs(sections: List[str]) -> re.Pattern:
+    return re.compile(r"\s*(" + "|".join(sections) + r"):")
 
-        if line.startswith("Fix:"):
-            risk_complete = True
-            continue
 
-        if line.startswith("Code for fix:"):
-            fix_complete = True
-            continue
+def process_paragraph(paragraph: str) -> str:
+    return dedent(paragraph)
 
-        if not summary_complete:
-            summary += line.strip()
-            summary += "\n"
-            continue
 
-        if not risk_complete:
-            risk += line.strip()
-            risk += "\n"
-            continue
-
-        if not fix_complete:
-            fix_text += line.strip()
-            fix_text += "\n"
-            continue
-
-        fix_code += line
-        fix_code += "\n"
-
-    return summary, risk, fix_text, fix_code
+def parse_doc_string(doc):
```

Comment:
@oscarbc96 could you include either a test for this method to show it parsing an incoming doc directly, or attach a screenshot showing you running this code locally and the documentation being formatted correctly :)

Reply:
@ocrawford555 this is how it looks 
![image](https://user-images.githubusercontent.com/4248262/71995222-8dfecf00-323a-11ea-81fc-87ceb28284eb.png)